CHANGELOG.md

```diff
@@ -9,6 +9,34 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.2.0 - 2019-12-18
+
+The Monitoring [CRDs] feature is graduated from Alpha to Beta.
+
+### Added
+
+- Add "Last Hearbeat Time" to [Octant] plugin to visualize the last time the Agent / Controller reported its status.
+- Add OVS version to Agent's monitoring CRD.
+- Add instructions to run [Octant] and the Antrea plugin either in a Pod or as a process. A Dockerfile and YAML manifest are included for deploying [Octant] as a Pod.
+- Support for GRE and STT tunnels for the Pod overlay network.
+
+### Changed
+
+- Use [libOpenflow] Go library to manage OVS flows instead of ovs-ofctl binary.
+- Minor changes to the OVS pipeline, in particular to fix issues when using the netdev OVS datapath type.
+- Officially-supported Go version is no longer 1.12 but 1.13.
+
+### Fixed
```

Comment:
Two more issues to consider, but not mandatory:
Issue #195 
Issue #200

Reply:
I don't want this file to turn into a git log dump, so I'm including this in "Minor changes to the OVS pipeline". Let me know if you think these deserve to be explicitly called out.

Reply:
it's ok to do as you suggest.